src/Introspecter.js

```diff
@@ -0,0 +1,104 @@
+import React, {useContext, useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import {Loading, useDataProvider} from 'react-admin';
+import ResourceSchemaAnalyzerContext from './ResourceSchemaAnalyzerContext';
+
+const IntrospecterComponent = ({
```

Comment:
bad name for a component. By the way, why does this file split its logic in two components that are entirely intricated?

Reply:
For testing purpose, IIRC.